SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+# SEP-0002 - SunPy Organization Definition
+
+# Abstract
+The SunPy project has changed significantly since the first version of SEP-0002 was written in 2014.
+This SEP details a new governance structure designed to enable the project to grow, while allowing for more input from the wider SunPy community.
+
+## Detailed Description
+The primary role of the SunPy Project is to *facilitate and promote the use and development of community-led, free and open-source solar data-analysis software based on the scientific Python environment*.
+This includes but is not limited to the following tasks:
+
+* developing the sunpy core library and other affiliated packages
+* manage and protect the SunPy brand and identity
+* promote SunPy to the community
+* managing SunPy assets, i.e. GitHub organization, domain name etc.
+
+# Governance Structure
+
+The SunPy project's governance involves the following five components:
+
+* The SunPy Community
+* The Project Members
+* The Steering Committee
+* The Advisory Board
+* The Ombudsperson
+
+The SunPy project is ultimately run by the whole community: users, developers, and members of the wider solar physics and scientific software community, for the benefit of all.
```

Comment:
```suggestion
The SunPy project is ultimately owned by the whole community: users, developers, and members of the wider solar physics and scientific software community, for the benefit of all.
```
It is simply not true that the entire community is running the day to day activities of the project.

Reply:
Isn't the project owned by the individuals who have direct access to the GitHub account?

Reply:
This has now been reworded.